libraries/botbuilder-ai/botbuilder/ai/luis/luis_prediction_options.py

```diff
@@ -9,7 +9,12 @@ class LuisPredictionOptions(object):
     Optional parameters for a LUIS prediction request.
     """
 
-    def __init__(self, timeout: float = 100000):
+    def __init__(
+        self,
+        timeout: float = 100000,
+        telemetry_client: BotTelemetryClient = NullTelemetryClient,
+        log_personal_information: bool = False,
+    ):
```

Comment:
How "to the letter of the law" are we going to be following C#?  I like this as a usability feature.
[https://github.com/Microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Builder.AI.LUIS/LuisPredictionOptions.cs](https://github.com/Microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Builder.AI.LUIS/LuisPredictionOptions.cs)